androidApp/app/src/main/java/com/example/ma/contextualawarenessapplication/BackgroundService.java

```diff
@@ -77,14 +97,78 @@ public void run() {
     @Override
     public void onDestroy() {
         Log.i(TAG,"onDestroy method called");
+        beaconManager.unbind(this);
     }
 
+
     @Override
     public IBinder onBind(Intent intent) {
         // TODO: Return the communication channel to the service.
         return null;
     }
 
+
+
+    @Override
+    public void onBeaconServiceConnect() {
+        beaconManager.setRangeNotifier(new RangeNotifier() {
+            @Override
+            public void didRangeBeaconsInRegion(Collection<Beacon> beacons, Region region) {
+                for (Beacon beacon : beacons) {
+                    MyBeacon myBeacon = new MyBeacon(beacon.toString());
+
+                    /*
+                    dbHandler.addBeacon(myBeacon);
+                    try {
+                        printDatabase();
+                    }catch (Exception e){
+                        Log.i("exxxx", e.toString());
+                    }
+                    */
+
+                    if(beacon.getDistance() < 1)
```

Comment:
what does the value of the distance represents. I suggest you create a static value with the representative name.